plugin/files/airconsole/airconsole-1.5.0.js

```diff
@@ -1,29 +0,0 @@
-/**
```

Comment:
A short explaination about this one... I got questionned about it...

JS is loaded from top to bottom (where logic browsers do cache and preload) and does load the next one when the previous did correctly execute. This script was injecting another link to a js script in the header, but sometimes too late as the script had already executed and the dom parsing was continuing already to the next script. Resulting in a random Undefined AirConsole() in the next script call, depending on the browser's cache, latency and other variables